src/relentless/simulate/hoomd.py

```diff
@@ -212,7 +212,30 @@ def _convert_to_gsd_file(self, sim):
                 gsd_filename = None
             gsd_filename = mpi.world.bcast(gsd_filename)
         else:
-            gsd_filename = self.filename
+            if mpi.world.rank_is_root:
+                gsd_filename = self.filename
+                with gsd.hoomd.open(self.filename) as snap:
+                    frame = snap[0]
```

Comment:
I would use `trajectory` or `t` or `f` here, based on usually conventions for opening these files:
```suggestion
                with gsd.hoomd.open(self.filename) as trajectory:
                    frame = trajectory[0]
```